demos/appyx-interactions/ios/src/iosMain/kotlin/main.ios.kt

```diff
@@ -0,0 +1,30 @@
+import androidx.compose.foundation.layout.BoxWithConstraints
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.window.ComposeUIViewController
+import theme.DarkColorPalette
+
+@Suppress("FunctionNaming")
+fun MainViewController() = ComposeUIViewController {
+    AppyxTheme {
+        BoxWithConstraints(
+            modifier = Modifier.fillMaxSize()
+        ) {
+            BackStackParallaxSample(
+                screenHeightPx = constraints.maxHeight,
+                screenWidthPx = constraints.maxWidth,
+                modifier = Modifier.fillMaxSize(),
+            )
+        }
+    }
+}
```

Comment:
I wonder if we should have the parallax sample, or the 3d stack sample, or the same exact interactions app as on android altogether?

Reply:
Guess it would be better to have the interactions app of Android. Would need to create a common module in :demos:appyx-interactions though. Will create a separate task for this.